js/game.js

```diff
@@ -1,27 +1,54 @@
 $(document).ready(function() {
-		var points = 0;
-	$('.picture').on('click', function() {
+	
+	var points = 0;
+
+	var isCorrectAnswer = function() {
```

Comment:
I was just talking to Annalise, and you would be better off naming `isCorrectAnswer` and `isWrongAnswer` to `correctAnswerListener` and `wrongAnswerListener`.